pydeseq2/dds.py

```diff
@@ -88,63 +98,49 @@ class DeseqDataSet:
 
     Attributes
     ----------
-    design_matrix : pandas.DataFrame
-        A DataFrame with experiment design information (to split cohorts).
-        Indexed by sample barcodes. Unexpanded, *with* intercept.
-
-    n_processes : int
-        Number of cpus to use for multiprocessing.
-
-    size_factors : pandas.Series
-        DESeq normalization factors.
-
-    non_zero_genes : pandas.Index
-        Index of genes that have non-uniformly zero counts.
-
-    genewise_dispersions : pandas.Series
-        Initial estimates of gene counts dispersions.
+    X
```

Comment:
What is the type of `X`?

Reply:
The type of attributes inherited from `anndata` is a little complex (c.f. [their doc](https://anndata.readthedocs.io/en/latest/generated/anndata.AnnData.html#anndata.AnnData)).

I suggest leaving this blank for now and filling the gap using [autodoc-typehints](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_typehints) once we've merged the type hinting PR (as is done in AnnData's documentation).

Reply:
OK!